styleguide/_layouts/default.jade

```diff
@@ -53,38 +53,38 @@ html(lang="en")
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Global Elements
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Global Theme
             ul.navZoolander-dropdown
               li.navZoolander-dropdownItem
-                a(href="derek/assets/").navZoolander-dropdownLink Overview
+                a(href="derek/globalTheme/").navZoolander-dropdownLink Overview
               li.navZoolander-dropdownItem
-                a(href="derek/assets/typography").navZoolander-dropdownLink Typography
+                a(href="derek/globalTheme/typography").navZoolander-dropdownLink Typography
               li.navZoolander-dropdownItem
-                a(href="derek/assets/photography").navZoolander-dropdownLink Photography/Banners
+                a(href="derek/globalTheme/photography").navZoolander-dropdownLink Photography/Banners
               li.navZoolander-dropdownItem
-                a(href="derek/assets/swatches").navZoolander-dropdownLink Colors/Swatches
+                a(href="derek/globalTheme/swatches").navZoolander-dropdownLink Colors/Swatches
               li.navZoolander-dropdownItem
-                a(href="derek/assets/icon-font").navZoolander-dropdownLink Icon Font
+                a(href="derek/globalTheme/icon-font").navZoolander-dropdownLink Icon Font
               li.navZoolander-dropdownItem
-                a(href="derek/assets/buttons").navZoolander-dropdownLink Buttons/CTAs
+                a(href="derek/globalTheme/buttons").navZoolander-dropdownLink Buttons/CTAs
               li.navZoolander-dropdownItem
-                a(href="derek/assets/tables").navZoolander-dropdownLink Tables
+                a(href="derek/globalTheme/tables").navZoolander-dropdownLink Tables
               li.navZoolander-dropdownItem
-                a(href="derek/assets/forms").navZoolander-dropdownLink Forms
+                a(href="derek/globalTheme/forms").navZoolander-dropdownLink Forms
 
           //Add in themes dropdown when there are multiple
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Derek Theme
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Ecommerce Theme
             ul.navZoolander-dropdown
               li.navZoolander-item
                 a(href="derek/").navZoolander-dropdownLink Overview
               li.navZoolander-item
-                a(href="derek/solutions").navZoolander-dropdownLink Solutions
+                a(href="derek/templates").navZoolander-dropdownLink Templates
               li.navZoolander-item
-                a(href="derek/resources").navZoolander-dropdownLink Resources
+                a(href="derek/patterns/patterns").navZoolander-dropdownLink Patterns
               li.navZoolander-item
-                a(href="derek/templates").navZoolander-dropdownLink Templates
+                a(href="derek/assetLibrary").navZoolander-dropdownLink Asset Library
```

Comment:
This should be `asset-library` for consistency with the naming conventions for other directories `styleguide/derek`. Wouldn't `assets` bo okay too?